FLEDGE.md

```diff
@@ -492,15 +492,21 @@ For the JSON response, only the `https` scheme is supported -- the `uuid-in-pack
 
 #### 2.5.2 Using Response Headers
 
-An alternative way to pass DirectFromSellerSignals without subresource bundles is via the `Ad-Auction-Signals` response header of some `fetch()` request, together with the `directFromSellerSignalsHeaderAdSlot` parameter on `navigator.runAdAuction()`.
+An alternative way to pass DirectFromSellerSignals without subresource bundles is via the `Ad-Auction-Signals` response header of some `fetch()` request or `iframe` navigation, together with the `directFromSellerSignalsHeaderAdSlot` parameter on `navigator.runAdAuction()`.
```

Comment:
Are there any timing requirements? For the fetch() approach, we need the promise to be resolved to ensure the headers have already arrived.

I assume the auction may be run / the directFromSellerSignalsHeaderAdSlot promise may be resolved immediately after the iframe DOM node with the correct src and attribute are inserted into the document tree (or maybe it doesn't even need to be inserted, just constructed)? (But, then how does the auction block on awaiting the network response?)

For static HTML that already has the <iframe> tag, does the document have to be in some loaded state (like OnLoad or some other event has happened)?

Reply:
Yes. I've added a couple of sentences here to emphasize those timing requirements. This echos what appears in the NOTE on the spec around when the Promises are expected to be resolved.